src/unstable-temp/DownloadImageModal/ImageForm.tsx

```diff
@@ -413,11 +413,9 @@ export const ImageForm = ({
 					onClick={(event: React.MouseEvent) => {
 						if (downloadOptions.provisioningKeyExpiryDate) {
 							// Convert to UTC for /download
-							downloadOptions.provisioningKeyExpiryDate = moment(
+							downloadOptions.provisioningKeyExpiryDate = parseISO(
```

Comment:
@myarmolinsky Sorry I didn't mention it here as well. would it also be a valid ISO string?

Reply:
@iamsolankiamit It works but we need to convert it to UTC too

Reply:
Nevermind it works, idk why it appeared to me that it did not at first ...

Reply:
Good catches, thank you 🚀